app/src/main/java/com/sdp/movemeet/view/profile/EditProfileActivity.java

```diff
@@ -109,8 +111,10 @@ private void assignViewsAndAdjustData() {
 
 
     public void changeProfilePicture(View view) {
-        Intent openGalleryIntent = new Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI);
-        startActivityForResult(openGalleryIntent, REQUEST_IMAGE);
+        if (isStorageReadPermissionGranted(this)) {
```

Comment:
Nice, thanks for adding this! As discussed, otherwise we could access the gallery to select a profile picture by default, without any permission...

Reply:
True, although I still find it weird that android studio didn't complain about the version without checks...